src/fides/api/ops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -167,6 +168,9 @@ def get_connections(
             ConnectionConfig.last_test_succeeded.is_(test_status.str_to_bool())
         )
 
+    if orphaned_from_system:
+        query = query.filter(ConnectionConfig.system_id.is_(null()))
```

Comment:
This could give unexpected results on `?orphaned_from_system=false`, I might make the default be None above, so you have three states:

If no query param value: all connections returned
If ?orphaned_from_system=true:  only connections with no system_id returned
If ?orphaned_from_system=false: only connections *with* a system id are returned

Can you add unit tests for this filter?

Reply:
Good idea. I'll add a unit test for this.